docs/management/admin/admin-pg-ov.asciidoc

```diff
@@ -17,25 +17,25 @@ NOTE: {fleet} must be enabled in a Kibana Space for administrative actions to fu
 
 The *Endpoints list* lists all hosts running {elastic-sec} and their relevant integration details. Endpoints appear in chronological order, with newly added endpoints at the top. The Endpoints list provides the following data:
```

Comment:
Use different verb to avoid repetition of "Endpoint list lists"? Maybe "The Endpoint list _displays_ all hosts..."